lib/plottr_components/src/components/dashboard/options/OptionsHome.js

```diff
@@ -251,6 +288,29 @@ const OptionsHomeConnector = (connector) => {
                 </div>
               ) : null}
             </Tab>
+            <Tab eventKey={5} title={t('Appearance')}>
+              <div className="dashboard__options__item">
+                <h4>{t('Default Font')}</h4>
+                <FontsButton
```

Comment:
Can we write bespoke components for selecting the font and font size in settings?
It feels wrong that you need to add two `useState` calls and create a dummy object for the editor props.